firestore-bigquery-export/firestore-bigquery-change-tracker/src/__tests__/bigquery/stresstest.test.ts

```diff
@@ -52,8 +54,68 @@ describe("Stress testing", () => {
 
       table = bq.dataset(datasetId).table(tableId_raw);
 
-      const rows = await table.getRows();
-      expect(rows[0].length).toEqual(100);
-    }, 120000);
+      const rows = await table.getRows({
+        selectedFields: "document_id",
+        maxResults: 100,
+      });
+      expect(rows[0].length).toEqual(count);
+    }, 340000);
+  });
+
+  describe("snapshot view stresstest", () => {
+    test("should run snapshot view query on a big table", async () => {
+      const query = buildLatestSnapshotViewQuery(
+        "new_stresstest",
+        "test_changelog_table",
+        "timestamp",
+        ["data", "operation", "event_id", "timestamp"],
+        "extensions-testing"
+      );
+
+      const [job] = await bq.createQueryJob({
+        query,
+        useLegacySql: false,
+      });
+
+      const [rows] = await job.getQueryResults();
+
+      expect(rows.length).toEqual(1);
+    }, 240000);
+    test("should get snapshot view query if some timestamps are null", async () => {
+      const query = buildLatestSnapshotViewQuery(
+        "new_stresstest",
+        "some_null",
+        "timestamp",
+        ["data", "operation", "event_id", "timestamp"],
+        "extensions-testing"
+      );
+
+      const [job] = await bq.createQueryJob({
+        query,
+        useLegacySql: false,
+      });
+
+      const [rows] = await job.getQueryResults();
+
+      expect(rows.length).toEqual(1);
+    }, 240000);
+    test("should get snapshot view query if duplicate timestamps exist", async () => {
+      const query = buildLatestSnapshotViewQuery(
+        "new_stresstest",
+        "duplicate_timestamp_table",
+        "timestamp",
+        ["data", "operation", "event_id", "timestamp"],
+        "extensions-testing"
+      );
+
+      const [job] = await bq.createQueryJob({
+        query,
+        useLegacySql: false,
+      });
+
+      const [rows] = await job.getQueryResults();
+
+      expect(rows.length).toEqual(1);
+    }, 240000);
   });
```

Comment:
As we are adopting a new SQL query for this task. Could we test the migration from the previous query, comparing the results to ensure that nothing has changed in the result set?

Reply:
ya can tie this into the other suggestion